ui/admin/app/templates/scopes/scope/sessions/index.hbs

```diff
@@ -114,7 +114,18 @@
                 </time>
               </row.cell>
               <row.cell>
-                <Rose::Badge>{{sessionAggregate.session.status}}</Rose::Badge>
+                {{#if sessionAggregate.session.isUnknown}}
+                  <SessionStatus @status={{sessionAggregate.session.status}} />
+                {{else}}
+                  <SessionStatus
+                    @status={{t
+                      (concat
+                        'resources.session.status.'
+                        sessionAggregate.session.status
+                      )
+                    }}
+                  />
+                {{/if}}
```

Comment:
This is potentially unsafe, since the translates strings differ by locale.  Let's pass in the session itself and do the known/unknown switching and translation within the component itself.  This simplifies any call sites so that we aren't performing this switch in multiple places.